src/Murder.Editor/Data/TexturePacker.cs

```diff
@@ -255,9 +255,10 @@ private void ScanForTextures(string path)
 
             FileInfo[] files = di.GetFiles("*.*", SearchOption.AllDirectories);
 
+            var ignoredExtensions = Architect.EditorSettings.IgnoredTexturePackingExtensions.Split(',');
             foreach (FileInfo fi in files)
             {
-                if (fi.Name.StartsWith("_"))
+                if (fi.Name.StartsWith("_") || ignoredExtensions.Contains(fi.Extension.ToLower()))
```

Comment:
We are shifting from `ToLower` to using a string comparison, which is safer on some platforms:
`ignoredExtensions.Contains(fi.Extension, StringComparer.InvariantCultureIgnoreCase))`